core/src/main/java/hudson/model/UsageStatistics.java

```diff
@@ -185,7 +185,7 @@ public String getStatData() throws IOException {
                 o.write(w);
             }
 
-            return new String(Base64.encode(baos.toByteArray()));
+            return new String(Base64.getEncoder().encode(baos.toByteArray()));
         } catch (GeneralSecurityException e) {
```

Comment:
Missing IllegalArgumentException ?

Reply:
ditto

Reply:
was this addressed?

Reply:
before of these changes, the IOException was not managed in any place, If I catch the IllegalArgumentException it will change the current behavior.

Reply:
it had not captured before the changes.